managed/services/management/node.go

```diff
@@ -121,9 +129,16 @@ func (s *NodeService) Register(_ context.Context, req *managementpb.RegisterNode
 			CreateNodeExporter(tx.Querier, pmmAgent.AgentID, nil, isPushMode(req.MetricsMode), req.DisableCollectors,
 				pointer.ToStringOrNil(req.AgentPassword), "")
 		return err
-	}); e != nil {
+	})
+	if e != nil {
 		return nil, e
 	}
 
+	apiKeyName := fmt.Sprintf("pmm-vmagent-%s-%d", req.NodeName, rand.Int63())
```

Comment:
```suggestion
	apiKeyName := fmt.Sprintf("pmm-agent-%s-%d", req.NodeName, rand.Int63())
```
it can be used for pmm-admin commands too